deployments/systemd/nvidia-mig-manager.service

```diff
@@ -15,12 +15,13 @@
 [Unit]
 Description=Configure MIG on NVIDIA GPUs
 DefaultDependencies=no
-After=sysinit.target local-fs.target
-Before=basic.target nvidia-persistenced.service systemd-resolved.service
+Wants=nvidia-persistenced.service nvidia-fabricmanager.service
+After=nvidia-persistenced.service nvidia-fabricmanager.service
```

Comment:
Just for my own understanding -- why is *both* `Wants` *and* `After` needed?

Reply:
I tested this again without the Wants and it seems to hold the order. Previously, I had seen an issue where nvsm was still starting before. I will remove the Wants

Reply:
I was also of the misunderstanding that they were effectively synonymous, but I can confirm Keith's findings and the the official documentation seems to line up with what we observed.

> Note that [the `Before=` and `After=` directives] are independent of and orthogonal to the requirement dependencies as configured by `Requires=`, `Wants=`, `Requisite=`, or `BindsTo=`. It is a common pattern to include a unit name in both the `After=` and `Wants=` options, in which case the unit listed will be started before the unit that is configured with these options.

https://www.freedesktop.org/software/systemd/man/latest/systemd.unit.html#Wants=
https://www.freedesktop.org/software/systemd/man/latest/systemd.unit.html#Before=